helm/hpcc/templates/_helpers.tpl

```diff
@@ -570,6 +570,8 @@ Generate instance queue names
   type: roxie 
   prefix: {{ .prefix | default "null" }}
   queriesOnly: true
+  replicas: {{ .replicas | default 2 }}
```

Comment:
Not convinced that this is right. Apart form anything else the default in roxie.yaml is 1.

I don't know if there's some way to make the default be only specified in one place...

Reply:
Actually, I don't think this replicas value has anything to do with what I want in the cluster info.  I think the cluster info is more data replica?  And in cloud we leave that up to the cloud environment?  I'll remove this setting from here.  And in the cluster info should probably just return 1.